app/assets/stylesheets/print.scss

```diff
@@ -25,4 +25,10 @@
   position: relative;
   top: 3px;
   font-family: HelveticaNeue, "Helvetica Neue", Arial, Helvetica, sans-serif;
+}
+
```

Comment:
The important flag thing always confuses me - I can see why we're not showing buttons when we print, but I can't work out the reason for the !important bit. Would a comment help?